app/src/main/kotlin/com/wire/android/ui/authentication/login/Login.kt

```diff
@@ -67,7 +75,11 @@ private fun LoginContent() {
                 Spacer(modifier = Modifier.height(16.dp))
                 ForgotPasswordLabel(modifier = Modifier.fillMaxWidth())
             }
-            LoginButton(modifier = Modifier.fillMaxWidth(), email = email.text, password = password.text)
+            LoginButton(modifier = Modifier.fillMaxWidth(), email = email.text, password = password.text) {
+                scope.launch {
+                    loginViewModel.navigateToConvScreen()
+                }
```

Comment:
I wonder if action like this should be scoped to a CoroutineScope of the Composable, I think making navigateToConvScreen not a suspend function and using ViewModel CoroutineScope would be a better approach

Reply:
I took `Home Drawer.kt` as inspiration, to me the navigation to a different screen is a UI task so scoping this to the `ViewModel` (which lives longer than the UI) makes no sense